android/app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <string name="persona_inquiry_start_title">Valora would like to verify your identity</string>
+  <string name="persona_inquiry_start_title">This app would like to verify your identity</string>
```

Comment:
wonder if this can somehow use app name. I guess its a barely used feature so maybe not too important